src/main/java/datawave/microservice/query/storage/QueryStatus.java

```diff
@@ -0,0 +1,435 @@
+package datawave.microservice.query.storage;
+
+import static datawave.microservice.query.storage.QueryStatus.QUERY_STATE.CLOSE;
+import static datawave.microservice.query.storage.QueryStatus.QUERY_STATE.CREATE;
+
+import java.io.ByteArrayOutputStream;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+import java.io.Serializable;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.accumulo.core.security.Authorizations;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import datawave.core.query.configuration.GenericQueryConfiguration;
+import datawave.core.query.logic.QueryKey;
+import datawave.microservice.authorization.user.DatawaveUserDetails;
+import datawave.microservice.query.Query;
+import datawave.microservice.querymetric.BaseQueryMetric.Prediction;
+import datawave.webservice.query.exception.DatawaveErrorCode;
+
+public class QueryStatus implements Serializable {
+    /**
+     * These are the possible query states correlating with the activity that the user is requesting. DEFINE: define a query CREATE: create a query and get
```

Comment:
small nit, the docs in class won't render in a list

Reply:
sorry, i don't follow.  what do you mean?

Reply:
i see what you mean.  working it

Reply:
should be good now.